### DISCUSSION TITLE
Tercümanlık Yapabilirim Sayfası Oluşturun

### DISCUSSION THREAD

**Discussion by sercangoger** [acikkaynak/deprem-yardim-projesi#36] [Ideas]
diğer ülkelerden gelen Profesyonel birliklere dil desteği verebilecek gönüllü arkadaşlarımız olmalı. fransızça bilen rusça bilen onlara destek verebilecek.
  yalnız kalmamaları onları yonetiminin kolaylaştırılması için.

**Comment by furkan-cat** [comment]
Rusça olarak Frontend Taraflı destek verebilirim.

**Comment by hasaneks** [comment]
Aslında sadece tercüman olarak değil biraz daha genel düşünüp gönüllü olabilirim gibi bir kategori açabilir miyiz ?

- Rehber
- Aşçı 
- Saha Görevlisi

Gibi birçok kategoriyi bir araya getirebiliriz. İlerleyen süreçte de STK'lar ile paylaşılır. Ben birçok STK'ya başvurdum ama tek bir havuz olması daha iyi olabilir.

deprem.io altına kategori olarak eklenebilir.

**Reply by sercangoger** [reply]
kesinlikle amaç havuz oluşturmak ve hızlı reaksiyon alabilmek. 100 ülkenin Profesyonel Ekipleri mevcut. bunlara bilgi aktarmak cok ama cok değerli ve onemli

**Reply by Arikitokurin** [reply]
Gönüllülükle birlikte afette islerini kaybeden insanlara da bir olanak sunsa?

**Reply by Arikitokurin** [reply]
![image](https://user-images.githubusercontent.com/124596763/218258470-4fc582c0-7e31-42f0-9d43-8ded0a1b3ff1.png)

**Reply by Arikitokurin** [reply]
![image](https://user-images.githubusercontent.com/124596763/218258588-48f7422d-7dcc-4d07-82e1-fab4cf646601.png)

**Comment by kivilcimcagatay** [comment]
çevbir ile işbirliği sağlanabilir orada çalışan arkadaşım var yardımcı olabilir

**Reply by sercangoger** [reply]
ozaman bu projeyi görmelerini sağlayalım hızlı bir şekilde yapılabilir.

**Comment by yamac-kurtulus** [comment]
Selam, şu an çevirmenlik konusunda bir proje ilerliyor bir taraftan, repo 1-2 saate public olur, buraya yazar arkadaşlar o noktada.

**Comment by abdullahsadikkuralay** [comment]
Merhaba, Boğaziçi Üniversitesi Çeviribilim öğrencileri adına yazıyorum. Sahada yabancı ekiplere destek veren çevirmenler için 18 dilde bir sözlük oluşturduk. Terimler Google Sheets'te çevirmenlerin çevrimdışı kullanabileceği, az şarj harcayan, yeni terimler eklendikçe sheetsten çekecek ve kullanıcı dostu arama özellikleri olan bir iOS ve Android uygulaması ihtiyacımız var nasıl destek alabiliriz?

**Comment by hakandilek** [comment]
Arkadaslar ben frontend kisminda hizlica yardimci olmaya calisabilirim. Fikrimce Progressive Web App olarak yapilirsa play store veya appsotore onayi beklemeden hizlica offline we mobilde calisabilecek bir uygulama olusturulabilir. Hatta Github Pages uzerinden bile ilk asamada yayina alinabilir. 

Ben offline calismaya basliyorum. @kilicfurkan bize repo olusturma ve organizasyon tarafinda yardimci olabilir misin?

**Reply by hakandilek** [reply]
Simdilik burdan basliyorum. Gerekli gorulurse tasiyabiliriz repoyu: https://github.com/hakandilek/afet-tercume

**Reply by yunusemrecetin** [reply]
@hakandilek Flutter ile geliştiriyoruz, offline çalışabilmesi için pwa desteği sağlıyor. Aynı sürede uygulama sürecini başlatabiliyoruz. Elimizde bir arayüzde mevcut

**Reply by hakandilek** [reply]
merhaba @yunusemrecetin, @xsorifc28'in asagida bahsettigi projede birliktesiniz sanirim. 
Az once @xsorifc28 ile discord uzerinde haberlestik. 

https://github.com/xsorifc28/tercuman-bulmaca kapsami tercumanlari insanlarla bulusturma uzerine, biz daha basit bir terimler sozlugu hedefliyoruz. @xsorifc28 ile simdilik hizlica bir seyler surebilmek icin calismalari bolmemeye karar verdik. ileride iki feature set birlestirilebilir.

**Comment by xsorifc28** [comment]
Selamlar biz bu projeye 2 gun once burada basladik ve 24 saat iceresinde teslim etmeye calisiyoruz:
https://github.com/xsorifc28/tercuman-bulmaca

@hakandilek

**Reply by hakandilek** [reply]
harikasiniz hocam. kolay gelsin!
